src/test/scala/com/feedzai/cosytest/core/ContainerIpSpec.scala

```diff
@@ -0,0 +1,46 @@
+package com.feedzai.cosytest.core
+
+import java.nio.file.Paths
+
+import com.feedzai.cosytest.{CleanUp, DockerComposeSetup, Utils}
+import org.scalatest.{FlatSpec, MustMatchers}
+
+import scala.concurrent.duration._
+import scala.util.Random
+
+class ContainerIpSpec extends FlatSpec with MustMatchers with CleanUp {
+
+  val setup = DockerComposeSetup(
+    Utils.randomSetupName,
+    Seq(Paths.get("src", "test", "resources", "docker-compose.yml")),
+    Paths.get("").toAbsolutePath,
+    Map.empty
+  )
+
+  override def dockerSetups = Seq(setup)
+
+  it should "Return None when container doesn't exist" in {
+    setup.getContainerIp("h64387g") mustBe None
+  }
+
+  it should "Return None for invalid container" in {
+    setup.up(2.minute)
```

Comment:
Why the special timeout?

Reply:
Actually there is no reason for that timeout. Just needed to give one :S